client/components/cards/attachments.js

```diff
@@ -1,5 +1,12 @@
 Template.attachmentsGalery.events({
-  'click .js-add-attachment': Popup.open('cardAttachments'),
+  'click .js-add-attachment'() {
+    Popup.open('cardAttachments').apply(this, arguments);
```

Comment:
Good advice, I will try and verify. 

BTW, some ES6 rules seems not working in my Chrome, like string template, weird.